packages/cli/src/commands/governance/propose.test.ts

```diff
@@ -174,9 +185,73 @@ testWithGanache('governance:propose cmd', (web3: Web3) => {
         '--jsonTransactions',
         'transactions.json',
         '--deposit',
-        '1000000000000000000',
+        '1000000000000000000', // TODO(Arthur): This is 1 CELO, shouldn't this be 10,000 CELO now?
+        '--from',
+        accounts[0],
+        '--descriptionURL',
+        'https://dummyurl.com',
+      ])
+
+      const proposal = await governance.getProposal(1)
+      expect(proposal.length).toEqual(transactions.length)
+      expect(proposal[0].to).toEqual(goldToken.address)
+      expect(proposal[0].value).toEqual(transactions[0].value)
+      const expectedInput = goldToken['contract'].methods['transfer'](
+        transactions[0].args[0],
+        transactions[0].args[1]
+      ).encodeABI()
+      expect(proposal[0].input).toEqual(expectedInput)
+    },
+    EXTRA_LONG_TIMEOUT_MS
+  )
+
+  test(
+    'will successfully create proposal based on Core contract with multisig (1 signer)',
+    async () => {
+      const transactionsToBeSaved = JSON.stringify(transactions)
+      fs.writeFileSync('transactions.json', transactionsToBeSaved, { flag: 'w' })
+
+      /*
+      TODO(Arthur): Why do we send the governance contract 1 CELO when
+      the `governance:propose` command makes a deposit with the the `--deposit` flag anyway?
+      Is this strictly necessary?
+      */
+      await (
+        await kit.sendTransaction({
```

Comment:
Note-to-self: Why do we send the governance contract 1 CELO? 
When we don't do this, the test fails, but I'm not 100% clear why. 
Perhaps the governance contract needs some CELO to execute governance related transactions?